docs/sql-migration-guide.md

```diff
@@ -105,13 +105,13 @@ license: |
 
   - Since Spark 3.1, CHAR/CHARACTER and VARCHAR types are supported in the table schema. Table scan/insertion will respect the char/varchar semantic. If char/varchar is used in places other than table schema, an exception will be thrown (CAST is an exception that simply treats char/varchar as string like before). To restore the behavior before Spark 3.1, which treats them as STRING types and ignores a length parameter, e.g. `CHAR(4)`, you can set `spark.sql.legacy.charVarcharAsString` to `true`.
 
-  - In Spark 3.1, `AnalysisException` is replaced by its sub-classes that are thrown for tables from Hive external catalog in the following situations:
+  - In Spark 3.1, `AnalysisException` is replaced by its sub-classes that are thrown for tables from Hive an external catalog in the following situations:
```

Comment:
I don't think `Hive external catalog` has any problem (`HiveExternalCatalog` class).

Reply:
+1 I'll close this.